### DISCUSSION TITLE
New maintainer: @nabokihms

### DISCUSSION THREAD

**Discussion by sagikazarmark** [dexidp/dex#1974] [General]
Hi everyone,

@nabokihms recently applied for becoming a maintainer of Dex. He has been an asset to the project in the last few months. He demonstrated an in-depth knowledge of Dex, contributed a bunch of changes (fixes, features) and helped others on the issue tracker. I vote for making him part of the team.

@dexidp/maintainers please vote 👍 / 👎  on this thread.

**Comment by justaugustus** [comment]
Sent a note to the mailing list: https://groups.google.com/u/1/g/dex-dev/c/Dsw2jsNhkWc
Lazy consensus timeout on Maksim's new maintainership set for Thursday, February 18th, EOD US Eastern.

**Comment by nate-double-u** [comment]
+1 NB

I'm not a maintainer, but want to share my support. @nabokihms has also helped out on the docs side and I think they'd be a great addition to the team.

**Comment by sagikazarmark** [comment]
@nabokihms welcome to the team!

**Reply by nabokihms** [reply]
Thank you all for your trust. I'm honored to be a part of the team.
